unknown_array_size.py

```diff
@@ -1,3 +1,10 @@
+"""
```

Comment:
As we are iterating through unlimited size array our approach is similar O(infinity) in linear search but in our approach it will be O(log(infinity))